modules/installation-network-user-infra.adoc

```diff
@@ -125,6 +129,24 @@ balancer after the bootstrap machine initializes the cluster control plane.
 |
 |Machine Config server
 
+|===
+
+
+2. *Application Ingress Load Balancer*: Provides an ingress point for application traffic flowing in from outside of the cluster.
+  * Layer 4 load balancing only (this is sometimes referred to as Raw TCP, SSL Passthrough, or SSL Bridge mode)
+  * Round robin load balancing
```

Comment:
`Round robin` is not always true in various cloud platform, for example the default distribution mode for Azure Load Balancer is a five-tuple hash. The tuple is composed of the: 
```
Source IP
Source port
Destination IP
Destination port
Protocol type
```
see also https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-distribution-mode

Reply:
Anyway, Round robin is most used load balancing algorithm. And both `round robin` and `hash` can used for Application Ingress LB.

Reply:
@etsauer - if you can address @lihongan comment, we can merge this.

@codyhoag

Reply:
@lihongan @vikram-redhat I'm not clear on what you are asking me to change here. Do you want me to just remove the `round robin` line since it doesn't apply across the board?

Reply:
@lihongan did you need @etsauer to do any update here?

Reply:
Since Round robin is not required here, so it is better to remove this line. Thanks.